elasticsearch/docker/config/docker-entrypoint.sh

```diff
@@ -46,17 +45,39 @@ done < <(env)
 # will run in.
 export ES_JAVA_OPTS="-Des.cgroups.hierarchy.override=/ $ES_JAVA_OPTS"
 
-# TODO: Replace supervisord with something else so we can certify the docker image (https://docs.docker.com/docker-hub/publish/certify-images/ says supervisord is forbidden)
-#if [[ -d "/usr/share/elasticsearch/plugins/opendistro-performance-analyzer" ]]; then
-#    CLK_TCK=`/usr/bin/getconf CLK_TCK`
-#    ES_JAVA_OPTS="-Dclk.tck=$CLK_TCK -Djdk.attach.allowAttachSelf=true $ES_JAVA_OPTS"
-#    if [[ -d "/usr/share/elasticsearch/performance-analyzer-rca" ]]; then
-#        ES_JAVA_OPTS="-Djava.security.policy=file:///usr/share/elasticsearch/performance-analyzer-rca/pa_config/es_security.policy $ES_JAVA_OPTS"
-#        /usr/bin/supervisord -c /usr/share/elasticsearch/performance-analyzer-rca/pa_config/supervisord.conf
-#    else
-#        ES_JAVA_OPTS="-Djava.security.policy=file:///usr/share/elasticsearch/plugins/opendistro-performance-analyzer/pa_config/es_security.policy $ES_JAVA_OPTS"
-#        /usr/bin/supervisord -c /usr/share/elasticsearch/plugins/opendistro-performance-analyzer/pa_config/supervisord.conf
-#    fi
-#fi
-
-/usr/share/elasticsearch/bin/elasticsearch "${es_opts[@]}"
+
+# Start up the elasticsearch and performance analyzer agent processes.
+# When either of them halts, this script exits, or we receive a SIGTERM or SIGINT signal then we want to kill both these processes.
+
+function terminateProcesses {
+    if kill -0 $ES_PID >& /dev/null; then
+        echo "Killing elasticsearch process $ES_PID"
+        kill -TERM $ES_PID
+        wait $ES_PID
+    fi
+    if kill -0 $PA_PID >& /dev/null; then
+        echo "Killing performance analyzer process $PA_PID"
+        kill -TERM $PA_PID
+        wait $PA_PID
+    fi
+}
```

Comment:
```
Redirecting Standard Output and Standard Error
       This construct allows both the standard output (file descriptor 1) and the standard error output (file descriptor 2) to be redirected to the file whose name is the expansion of word.

       There are two formats for redirecting standard output and standard error:

              &>word
       and
              >&word

       Of the two forms, the first is preferred.  This is semantically equivalent to

              >word 2>&1
```

In bash man page, it states clear that you implementation of `>& /dev/null` will redirect stdout and stderr to /dev/null, which is not the same as remove stdout but reserving stderr. Suggest using `> /dev/null`.

Reply:
My mistake, I do, in fact, want to redirect both stderr and stdout. Sorry for the confusion!